src/main/java/core/basesyntax/io/write/impl/csv/CsvWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.io.write.impl.csv;
+
+import core.basesyntax.io.write.ReportWriter;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvWriterImpl implements ReportWriter {
+    @Override
+    public void write(String report, String reportPath) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(reportPath))) {
+            bufferedWriter.write(reportPath);
+        } catch (IOException e) {
+            throw new RuntimeException("Failed to write to a file");
```

Comment:
According to the checklist, you should rethrow a RuntimeException with an informative message and exception object. Include the reportPath in the message and pass the caught IOException as a cause to the RuntimeException.